src/worklets/numbers.ts

```diff
@@ -11,3 +11,19 @@ export function clamp(value: number, lowerBound: number, upperBound: number): nu
   'worklet';
   return Math.min(Math.max(lowerBound, value), upperBound);
 }
+
+/**
+ * Returns a random integer between min (inclusive) and max (inclusive).
+ * The value is no lower than min (or the next integer greater than min
+ * if min isn't an integer) and no greater than max (or the next integer
+ * lower than max if max isn't an integer).
+ * Using Math.round() will give you a non-uniform distribution!
+ */
+export function getRandomInt(min: number, max: number): number {
+  'worklet';
+  // eslint-disable-next-line no-param-reassign
+  min = Math.ceil(min);
+  // eslint-disable-next-line no-param-reassign
+  max = Math.floor(max);
```

Comment:
Is there a reason to reassign these args? Generally would avoid this unless that's the explicit purpose of the function, to mutate the inputs

Reply:
I had just moved this function from a different file, but agreed.